.github/workflows/api-test-lint-deploy.yaml

```
@@ -120,6 +120,8 @@ jobs:
         run: make -C api test-cov
       - name: Ensure assets build
         run: make -C api sdist wheel
+      - name: Check labware stacking regression tests
```

```suggestion
      - name: Check stacking regression tests
```
Kind of a future proofing nit, but since we'll probably add gripper stacking tests shortly, can we rephrase this?